pages/screening_subject_search/advance_fobt_screening_episode_page.py

```diff
@@ -28,6 +28,15 @@ def __init__(self, page: Page):
         self.record_other_post_investigation_contact_button = self.page.get_by_role(
             "button", name="Record other post-"
         )
+        self.enter_diagnostic_test_outcome_button = self.page.get_by_role(
+            "button", name="Enter Diagnostic Test Outcome"
+        )
+        self.handover_into_symptomatic_care_button = page.get_by_role(
+            "button", name="Handover into Symptomatic Care"
+        )
+        self.record_diagnosis_date_button = page.get_by_role(
+            "button", name="Record Diagnosis Date"
+        )
```

Comment:
Add self. in front of page
```suggestion
        self.handover_into_symptomatic_care_button = self.page.get_by_role(
            "button", name="Handover into Symptomatic Care"
        )
        self.record_diagnosis_date_button = self.page.get_by_role(
            "button", name="Record Diagnosis Date"
        )
```